hack/e2e-suite/services.sh

```diff
@@ -59,7 +59,6 @@ function join() {
 svcs_to_clean=()
```

Comment:
Not affected by your change, but for my own understanding, does anything ever actually get added to this?